### ISSUE TITLE
Method for uploading a calculation on the GUI from a file

### ISSUE THREAD

**Issue by cjh1** [OpenChemistry/mongochemclient#17]
@cryos Does this want to be launched from a molecule, so we know which molecules to associate the calculation with?

**Comment by cryos** [comment]
Yes, or enable upload and automatically create the molecule. From molecule first off would be fine, but it shouldn't be much work to extend that to create a new molecule/find an existing molecule.
